modules/sildisco/src/Auth/Process/TrackIdps.php

```diff
@@ -17,7 +20,7 @@ class TrackIdps extends \SimpleSAML\Auth\ProcessingFilter
     public function process(array &$state): void
     {
         // get the authenticating Idp and add it to the list of previous ones
-        $session = \SimpleSAML\Session::getSessionFromRequest();
+        $session = Session::getSessionFromRequest();
```

Comment:
Thinking about Session... perhaps a "use ... as SS_Session" might be useful, given there is a Yii session object.